File: android/src/main/kotlin/com/syncrhrohealth/bandpass_ble_connection/core/command/BasicInfoGetCmd.kt
package com.syncrhrohealth.bandpass_ble_connection.core.command

import com.syncrhrohealth.bandpass_ble_connection.core.command.builder.CommandBuilder
import com.syncrhrohealth.bandpass_ble_connection.core.enum.CommandCode
import com.syncrhrohealth.bandpass_ble_connection.core.handler.DeviceHandler


object BasicInfoGetCmd {
    fun send(handler: DeviceHandler) {
        val builder = CommandBuilder.build(CommandCode.PACKET_CMD_BASIC_INFO_GET)
        handler.write(builder)
    }
}

File: ios/Classes/Core/Command/BasicInfoGetCmd.swift
//
//  BasicInfoGetCmd.swift
//  Runner
//
//  Created by MAC on 11/3/25.
//

import Foundation

struct BasicInfoGetCmd {
    static func send(handler: DeviceHandler) {
        let builder = CommandBuilder.build(commandCode: .packetCmdBasicInfoGet)
        handler.write(data: builder)
    }
}
